test/TestProjects/TypeSchemaMapping/SomeFolder/Generated/Models/RenamedModelStruct.cs

```diff
@@ -52,12 +51,8 @@ namespace CustomNamespace
         /// <param name="propertyToField"> . </param>
         /// <param name="fruit"> Fruit. </param>
         /// <param name="daysOfWeek"> Day of week. </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="customizedFlattenedStringProperty"/> or <paramref name="propertyToField"/> is null. </exception>
         public RenamedModelStruct(string customizedFlattenedStringProperty, string propertyToField, CustomFruitEnum? fruit, CustomDaysOfWeek? daysOfWeek)
         {
-            Argument.AssertNotNull(customizedFlattenedStringProperty, nameof(customizedFlattenedStringProperty));
```

Comment:
lost assertions?

Reply:
This is due to the difference in validation handling. In DPG, we don't validate optional fields, and if I recall correctly, we agreed to remove this validation from HLC, and from what I can tell, there are no such cases in azure-sdk-for-net - only in tests.